sim/riscv/sim-main.c

```diff
@@ -1286,13 +1286,19 @@ execute_i (SIM_CPU *cpu, unsigned_word iw, const struct riscv_opcode *op)
   unsigned_word sb_imm = EXTRACT_SBTYPE_IMM (iw);
   unsigned_word shamt_imm = ((iw >> OP_SH_SHAMT) & OP_MASK_SHAMT);
   unsigned_word tmp;
+  unsigned_word sys_id;
+  int eh_rve_p = cpu->elf_flags & 0x8;
```

Comment:
Should use an EF_* value from include/elf/riscv.h, except that one hasn't been defined for RVE. So probably should define one first.

Reply:
Yeah, I think the problem is RVE support not merge to riscv-binutils-2.29 yet, I'll remove RVE in this PR and send another for this.